packages/odyssey/src/scss/components/_tag.scss

```diff
@@ -17,11 +17,13 @@
   margin: 0 $spacing-xs $spacing-xs 0;
   padding: 0 $spacing-xs-em;
   border-radius: $base-border-radius;
-  background: cv('gray', '200');
+  background: cv('gray', '100');
   color: $text-body;
 }
 
 .ods-tag--list {
+  display: inline-block;
```

Comment:
I don't think we should make these changes to the component unless we're also adding `.ods-tag--label` as mandatory for Inline lists or providing two variant styles. Currently, this'll add that margin regardless of how it's being used.